README.md

```diff
@@ -21,7 +21,74 @@ This is a Lambda Labs Capstone Project designed for rental property management.
 	* Automatically generate guest invoices
 	* Allow property managers to add new properties, assign properties to employees, and create and edit task checklists for employees
 	* Allow guests to view readiness of their assigned rental property and pay their invoice online
+
+## Tech Stack
+
+### Frontend built using:
+#### React
+
+With its vast eco-system and modular nature React is a great choice for developing the front-end of our project.
+
+	* The associated documentation, community and 3rd party packages are excellent
+	* It allows for reusability of already written components which reduces development time
+	* It is a tried and tested framework that will help us avoid sudden shifts in the technology causing major refactors
+
+#### React Hooks
+
+React Hooks simplify working with React due to use near exclusive use of functional components versus class components. Also, the use of React hooks results in smaller package sizes.
```

Comment:
Might be worth expounding on the virtues of hooks, like what smaller package sizes does for the app.